templates/palletInfo.html

```diff
@@ -4,8 +4,16 @@
 {% endblock %}
 {% block content %}
 <br/>
-    <button type="button" class="btn btn-success btn-lg" onclick="goBack()">&#8592;Back</button>
-    <h2>Location: {{ pallet_location_id }}</h2>
+    {% set location = request.args.get('location').split('-') %}
+    {% if location|length == 2 %}
+        <a href="{{ url_for('location_page', location_id=location[0], rack_id = location[1] )}}" 
+        class="btn btn-success btn-lg" tabindex="-1" role="button" aria-disabled="true">&#8592;Back</a>
+    {% else %}
+        <a href="{{ url_for('location_page', location_id=location[0], )}}" 
+        class="btn btn-success btn-lg" tabindex="-1" role="button" aria-disabled="true">&#8592;Back</a>
```

Comment:
look up font awesome for some icons if youre interested. might look better than html characters. [Font Awesome](https://fontawesome.com/)